src/zenml/integrations/great_expectations/__init__.py

```diff
@@ -35,6 +35,7 @@ class GreatExpectationsIntegration(Integration):
         "great-expectations>=0.15.0,<=0.15.47",
         # typing_extensions 4.6.0 and above doesn't work with GE
         "typing_extensions<4.6.0",
+        "sqlmodel<0.0.10",
```

Comment:
should add a comment on why we've added sqlmodel here. and maybe a note in the docs that this is what's happening?